x-pack/plugins/cloud_security_posture/public/pages/rules/rules_table.tsx

```diff
@@ -124,12 +136,36 @@ const getColumns = ({
     field: 'updatedAt',
     name: TEXT.UPDATED_AT,
     width: '15%',
-    render: timestampRenderer,
+    render: (timestamp) => moment.duration(moment().diff(timestamp)).humanize(),
   },
   {
     field: 'attributes.enabled',
     name: TEXT.ENABLED,
-    render: createRuleEnabledSwitchRenderer({ toggleRule }),
+    render: (enabled, rule) => (
+      <EuiToolTip
+        content={
+          enabled ? (
+            <FormattedMessage
+              id="xpack.csp.rules.deactivateRuleLabel"
+              defaultMessage="Deactivate Rule"
+            />
+          ) : (
+            <FormattedMessage
+              id="xpack.csp.rules.activateRuleLabel"
+              defaultMessage="Activate Rule"
+            />
+          )
+        }
+      >
+        <EuiSwitch
+          showLabel={false}
+          label={enabled ? TEXT.DISABLE : TEXT.ENABLE}
```

Comment:
is label needed even when `showLabel={false}`?

Reply:
nice catch! i don't think that's needed 👍

Reply:
![Screen Shot 2022-03-16 at 13 32 08](https://user-images.githubusercontent.com/20814186/158580850-d4c8e7db-481a-412a-8f3e-d1bf1d9b78c8.png)

turns out it is
might be an issue with upstream types